src/test/regress/expected/multi_fix_partition_shard_index_names.out

```diff
@@ -8,6 +8,13 @@ SET citus.next_shard_id TO 910000;
 SET citus.shard_replication_factor TO 1;
 CREATE SCHEMA fix_idx_names;
 SET search_path TO fix_idx_names, public;
+SELECT stop_metadata_sync_to_node('localhost', :worker_1_port);
```

Comment:
I guess we stop metadata sync to make sure that the tests work as intended (e.g., from non-metadata synced to metadata sync cluster such as upgrades)? 
If that is the case, I think it makes sense. But, would be good to have a comment on why we stop metadata at this point.

And, to prevent flaky test outputs make sure to run this test sequentially (e.g., no parallel tests) if that's not already the case.

Reply:
I'm adding the comment and yes there is no parallel test.